api/output/Json.php

```diff
@@ -20,6 +20,7 @@ public function printHeader()
     {
         header('Access-Control-Allow-Origin: *');
         header('Content-Type: application/json;charset=UTF-8');
+        header('Cache-Control: public, max-age=15');
```

Comment:
But... this is not always the case, is it? We probably should only set an etag header? Going to read up on HTTP caching strategies where cache of certain resources can be invalidated by user input...